src/resdk/resolwe.py

```diff
@@ -495,7 +503,20 @@ def _download_files(self, files, download_dir=None):
 
                 self.logger.info("* %s", os.path.join(file_path, file_name))
 
-                with open(
+                file_directory = os.path.dirname(file_url)
+                if file_directory not in sizes:
+                    sizes[file_directory] = {
+                        entry["name"]: entry["size"]
+                        for entry in json.loads(
+                            self.session.get(file_directory, auth=self.auth).content
+                        )
+                        if entry["type"] == "file"
+                    }
+                file_size = sizes[file_directory][file_name]
+
+                with tqdm.tqdm(
```

Comment:
Would be nice to show the name of the file that is being downloaded. Use:
`tqdm(... desc=f"Downloading file {file_name}")`
or something similar.

I realize that there are some `logger.info` calls, but AFAIK, most of the users does not turn them on.

Reply:
Where?